src/components/ArtifactCard.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <p-card class="artifact-card" :class="classes.root">
+    <div class="artifact-card__body" :class="classes.body">
+      <header class="artifact-card__header" :class="classes.header">
+        <h6 class="artifact-card__subheader" :class="classes.subheader">
+          {{ artifact.type }}
+        </h6>
+        <h3 v-if="hasKey">
+          {{ artifact.key }}
+        </h3>
+
+        <template v-if="hasRun && crumbs.length">
+          <p-bread-crumbs :crumbs="crumbs" class="artifact-card__bread-crumbs" :class="classes.breadCrumbs" />
+        </template>
+      </header>
+
+      <div class="artifact-card__summary-container" :class="classes.summaryContainer">
+        <div
+          class="artifact-card__summary-item"
+          :title="artifact.updated.toLocaleString()"
+        >
+          <span class="artifact-card__summary-item-label">
+            {{ localization.info.lastUpdated }}
+          </span>
+          <span class="artifact-card__summary-item-value">
+            {{ formatDateTime(artifact.updated) }}
+          </span>
+        </div>
+
+        <div
+          class="artifact-card__summary-item"
+          :title="artifact.created.toLocaleString()"
+        >
+          <span class="artifact-card__summary-item-label">
+            {{ localization.info.created }}
+          </span>
+          <span class="artifact-card__summary-item-value">
+            {{ formatDateTime(artifact.created) }}
+          </span>
+        </div>
+      </div>
+    </div>
+
+    <div v-if="slots.default" class="artifact-card__slot">
+      <slot />
+    </div>
+  </p-card>
+</template>
+
+<script lang="ts" setup>
+  import { BreadCrumbs } from '@prefecthq/prefect-design'
+  import { computed, useSlots } from 'vue'
+  import { useRoute } from 'vue-router'
+  import { useFlowRun, useTaskRun, useWorkspaceRoutes } from '@/compositions'
+  import { localization } from '@/localization'
+  import { Artifact } from '@/models'
+  import { formatDateTime } from '@/utilities'
+
+  const props = defineProps<{
+    artifact: Artifact,
+    condense?: boolean,
+  }>()
+
+  const slots = useSlots()
+  const routes = useWorkspaceRoutes()
+  const route = useRoute()
+
+  const flowRunId = computed(() => props.artifact.flowRunId)
+  const taskRunId = computed(() => props.artifact.taskRunId)
+
+  const flowRun = useFlowRun(flowRunId)
+  const taskRun = useTaskRun(taskRunId)
+
+  const hasRun = computed(() => !!props.artifact.flowRunId || !!props.artifact.taskRunId)
+  const hasKey = computed(() => !!props.artifact.key)
+
+  const crumbs = computed<BreadCrumbs>(() => {
```

Comment:
So with flow runs when we show crumbs we omit the flow if we're on the flow page. But here you're doing the opposite. 

I'm probably missing some context. But won't all the artifacts have the same crumbs?

Reply:
Ah maybe this isn't as readable as it should be but the idea is exactly the same: don't show crumbs for pages we're on. Starting with no crumbs, we check if the flow run exists and that its ID isn't in the current route:
```ts
    if (flowRun.value && !routeIncludes.value.flowRun) {
      internalCrumbs.push({
        text: flowRun.value.name ?? '',
        to: routes.flowRun(flowRun.value.id),
      })
    }
```

Reply:
On the flow run page, for instance, all the flow run crumbs would be the same (and so they would be omitted) but the task run crumbs would be different

Reply:
Ah okay that makes sense. I was just reading this wrong. 

I might break up the `routeIncludes` computed into two checks that are similar to `const viewingFlowRun = computed(() => ...)`. Might make it a little more readable

Reply:
Makes sense - modified that